.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
```

Comment:
The flake8 and isort configs have different line lengths. I'd vote for 80 chars.

Reply:
Oops, this identified that I'm missing flake8-bugbear. Let me fix that first...

Reply:
Okay, now that I've added flake8-bugbear to the requirements, this will make more sense.

This is non-obvious, but the flake8 config is actually handled with a bit more nuance, because it's based on black's reasoning for using 88 chars:
https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

> If you’re using Flake8, you can bump max-line-length to 88 and forget about it. Alternatively, use Bugbear’s B950 warning instead of E501 and keep the max line length at 80 which you are probably already using.

This leads to Bugbear's B950:
https://github.com/PyCQA/flake8-bugbear#opinionated-warnings
> B950: Line too long. This is a pragmatic equivalent of pycodestyle's E501: it considers "max-line-length" but only triggers when the value has been exceeded by more than 10%. You will no longer be forced to reformat code due to the closing parenthesis being one character too far to satisfy the linter. At the same time, if you do significantly violate the line length, you will receive a message that states what the actual limit is. This is inspired by Raymond Hettinger's "Beyond PEP 8" talk and highway patrol not stopping you if you drive < 5mph too fast. Disable E501 to avoid duplicate warnings.

The flake8 config disables E501 (the hard limit on line length) and instead uses a 10% buffer from B950, which matches isort's 88 lines and black's default 88 lines:

The functional distinction here is that 80 chars is the official limit, but it won't cause black (or isort) to break formatting in order to accommodate it.